ogusa/tests/test_user_inputs.py

```diff
@@ -2,7 +2,7 @@
 import os
 from ogusa.execute import runner
 CUR_PATH = os.path.abspath(os.path.dirname(__file__))
-PUF_PATH = os.path.join(CUR_PATH, '../puf.csv')
+PUF_PATH = os.path.join(CUR_PATH, '..', 'puf.csv')
```

Comment:
@jdebacker perhaps, this is the change that's throwing off the code coverage reporting?